charts/tekton-pipelines/templates/buildpacks/pipelines/_buildpack.yaml

```diff
@@ -162,6 +176,10 @@ spec:
       runAfter:
         - kustomize
 
+  {{ if .sentry.enabled }}
```

Comment:
I think that is smart - you do it this way to avoid getting artifacts in the repository?

Reply:
ok!